src/components/AccountDetailsPanel.vue

```diff
@@ -9,12 +9,13 @@
       </app-chip>
 
       <copy-chip
-        :label="accountDetails.id"
-        class="account-details-panel__copy-chip"/>
+        :label="formatKnownAddress(accountDetails.id, false)"
+        :clipboard-text="accountDetails.id"
+        class="u-hidden-mobile"/>
       <copy-chip
-        :label="formatEllipseHash(accountDetails.id)"
+        :label="formatKnownAddress(accountDetails.id)"
```

Comment:
I touched this and noticed the usage of styling classes for the different views, which I recently replaced. Here was forgotten, so I adjusted it also in sibling files